bundle/src/define/define-slot.ts

```diff
@@ -209,13 +209,29 @@ const defineSlot = (
 		// wait until IAS has initialised before checking teads eligibility
 		const isTeadsEligible = isEligibleForTeads(id);
 
-		slot.setConfig({
+		/*
+			eslint-disable-next-line @typescript-eslint/no-unnecessary-condition --
+			the slot.setConfig function may not exist if googletag has been shimmed by an adblocker
+		*/
+		slot.setConfig?.({
```

Comment:
Could we move this below the if statement to avoid the need for a conditional operator here?